docs/1.concepts/account-abstraction/chainsig.md

```diff
@@ -0,0 +1,124 @@
+---
+id: chain-signatures
+title: Chain Signatures
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+
+Chain Signatures allows you to call the `sign()` method on `multichain.near`:
```

Comment:
I want to add more context to what chain signatures are here. Only having that they allow you to call a method on a contract isn't super helpful here. I've started a doc to help with this here @bucanero: https://docs.google.com/document/d/1Rxwg24SX6zT1VKdUkieeCHLlvUcJwUPHuC21XG1gUrQ

Reply:
Thanks for sharing, I've updated the draft and added more context about Chain signatures. 👍